devlib/instrument/parse_aep.py

```diff
@@ -0,0 +1,509 @@
+#!/usr/bin/env python
```

Comment:
As mentioned above, this probably belongs in  the same file the instrument. But even if this does end up in a separate module, `devlib/instrument/` should *only* contain instruments.  This would belong under `devlib/utils/`

Reply:
ok. i'm going to have a look